### PR TITLE
Download & unzip theme from dl.gitea.io

### PR DESCRIPTION
**Pull request by metalmatze**
We know have a working theme at https://dl.gitea.io/theme/master.tar.gz. We can simply download & unzip into the `theme/` folder.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,7 +13,8 @@ build: $(THEME)
 
 .PHONY: update
 update: $(THEME)
```
Comment (tboerger) [inline Makefile:21]: I think we can drop this task now. Maybe we should also add a clean task and put the url into a variable than you can overwrite it locally or even later on for different branches within drone with an environment variable.

Comment (metalmatze) [reply Makefile:21]: Yep. I agree. Just one question am I supposed to type `make theme/gitea`? Maybe we should simply use `make theme` or keep update and delete theme/gitea.

Comment (tboerger) [reply Makefile:21]: You just need to execute initials make server or make build. If you want to update the theme just execute make clean server or make clean build.

**Comment by tboerger** [comment]
LGTM

**Review by tboerger — APPROVED** [review]

**Comment by Bwko** [comment]
LGTM

**Pull request merged**